### PR TITLE
Register proxy and beanstalkd into conscience

### PR DESCRIPTION
**Pull request by murlock**
##### SUMMARY
Register `proxy` and `beanstalkd` into `conscience`, it will allow us to launch commands on all cluster.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
conscience-agent

##### SDS VERSION
```
master
```

### CODE REVIEW

File: oio/conscience/checker/beanstalk.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                raise exc.ConfigurationException(
+                    'Missing field "%s" in configuration' % k)
+        print("YYY")
+
+    def check(self):
+        result = False
+        cnx = None
+        try:
+            cnx = Beanstalk(self.checker_conf['host'], self.checker_conf['port'])
+            cnx.stats_tube(self.checker_conf['tube'])
```
Comment (fvennetier) [inline oio/conscience/checker/beanstalk.py]: I think we only need to check the state of the service, not a specific tube. Calling `stats` would be better (must be implemented in `beanstalk.py`).

File: tools/oio-bootstrap.py [hunk 2]
```diff
@@ -426,6 +426,32 @@ def save_coverage():
...
+host: ${IP}
+port: ${PORT}
+type: beanstalkd
+location: ${LOC}
+checks:
+    - {type: beanstalk, tube: oio}
+slots:
+    - beanstalk
+stats:
+    - {type: system}
```
Comment (fvennetier) [inline tools/oio-bootstrap.py]: Beanstalkd services store data on the file system, thus we should monitor the volume they are using.
```suggestion
    - {type: system}
    - {type: volume, path: ${VOLUME}}
```

File: tools/oio-bootstrap.py [hunk 3]
```diff
@@ -426,6 +426,32 @@ def save_coverage():
...
 
+template_beanstalk_watch = """
+host: ${IP}
+port: ${PORT}
+type: beanstalkd
+location: ${LOC}
+checks:
+    - {type: beanstalk, tube: oio}
+slots:
+    - beanstalk
```
Comment (fvennetier) [inline tools/oio-bootstrap.py]: To remove any ambiguity, we should write beanstalk**d** everywhere.

Comment (murlock) [reply tools/oio-bootstrap.py]: :+1:

File: tools/oio-bootstrap.py (hunk 2 above)
Comment (murlock) [reply tools/oio-bootstrap.py]: Done

File: tools/oio-bootstrap.py [hunk 4]
```diff
@@ -426,6 +426,32 @@ def save_coverage():
...
+host: ${IP}
+port: ${PORT}
+type: beanstalkd
+location: ${LOC}
+checks:
+    - {type: beanstalkd}
+slots:
+    - beanstalkd
+stats:
+    - {type: system, path: ${VOLUME}}
```
Comment (fvennetier) [inline tools/oio-bootstrap.py]: Still not good.
```suggestion
    - {type: system}
    - {type: volume, path: ${VOLUME}}
```

**Pull request merged**